compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/antlr4/BallerinaParserBaseListener.java

```diff
@@ -1,4 +1,4 @@
-// Generated from BallerinaParser.g4 by ANTLR 4.5.3
+// Generated from /home/chiran/Desktop/WSO2/Ballerina/ballerina-lang/compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4 by ANTLR 4.5.3
```

Comment:
Shall we remove the 

/home/**chiran**/Desktop/WSO2/Ballerina/ballerina-lang/compiler/ballerina-lang/src/main/resources/grammar/ 

part?